src/core/projectinfo.cpp

```diff
@@ -16,11 +16,14 @@
 
 
 #include "projectinfo.h"
+#include "qgismobileapp.h"
```

Comment:
What is this used for?
I'd like to split off qgismobileapp.h into app sooner or later, other things shouldn't depend on it

Reply:
@m-kuhn , actually I was using it to get the static SUPPORTED_PROJECT_EXTENSIONS string list, but since then changed the way I went at it, removing.

Reply:
@m-kuhn , header removed.